Source/Lib/Codec/EbResourceCoordinationProcess.c

```diff
@@ -563,35 +563,34 @@ void* ResourceCoordinationKernel(void *inputPtr)
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->croppingRightOffset = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padRight;
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padBottom = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->maxInputPadBottom;
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->croppingBottomOffset = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padBottom;
-
-                if(contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padRight != 0 || contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padBottom != 0) {
-                    contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->conformanceWindowFlag = 1;
+            if(contextPtr->sequenceControlSetInstanceArray[instanceIndex]->encodeContextPtr->initialPicture) {
+                sequenceControlSetPtr->lumaWidth    = sequenceControlSetPtr->maxInputLumaWidth;
+                sequenceControlSetPtr->lumaHeight   = sequenceControlSetPtr->maxInputLumaHeight;
+                sequenceControlSetPtr->chromaWidth  = (sequenceControlSetPtr->maxInputLumaWidth >> subWidthCMinus1);
+                sequenceControlSetPtr->chromaHeight = (sequenceControlSetPtr->maxInputLumaHeight >> subHeightCMinus1);
+
+                sequenceControlSetPtr->padRight             = sequenceControlSetPtr->maxInputPadRight;
+                sequenceControlSetPtr->croppingRightOffset  = sequenceControlSetPtr->padRight;
+                sequenceControlSetPtr->padBottom            = sequenceControlSetPtr->maxInputPadBottom;
+                sequenceControlSetPtr->croppingBottomOffset = sequenceControlSetPtr->padBottom;
+
+                if(sequenceControlSetPtr->padRight != 0 || sequenceControlSetPtr->padBottom != 0) {
+                    sequenceControlSetPtr->conformanceWindowFlag = 1;
                 }
                 else {
-                    contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->conformanceWindowFlag = 0;
+                    sequenceControlSetPtr->conformanceWindowFlag = 0;
                 }
-                
-				inputSize = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->lumaWidth * contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->lumaHeight;
+
+                sequenceControlSetPtr->encoderBitDepth = sequenceControlSetPtr->staticConfig.encoderBitDepth;
+
+				inputSize = sequenceControlSetPtr->lumaWidth * sequenceControlSetPtr->lumaHeight;
```

Comment:
line up with line above

Reply:
Thanks for your suggestion. Tabs here have been convert to spaces in the new commit.